source/2020-01-17-the-ember-times-issue-131.md

```diff
@@ -19,6 +21,33 @@ READMORE
 
 ---
 
+## [4 RFCs on improving Ember's autotracking and reactivity system 💬](https://github.com/emberjs/rfcs/blob/use-and-resources/text/0567-use-and-resources.md#introducing-use-and-resources)
+
+Ember Octane features a **new reactivity system**. In particular, **tracked properties** simplify syncing the DOM with JavaScript changes. (Find out how autotracking works from the [Ember Guides](https://guides.emberjs.com/release/in-depth-topics/autotracking-in-depth/).)
```

Comment:
I would remove the brackets around such a long sentence ;) also because I think it is an important sentence

Reply:
Ok, sounds good.